contents/docs/migrate/migrate-to-cloud.mdx

```diff
@@ -67,11 +93,19 @@ python3 ./migrate.py \
    --cursor the-cursor-value-from-the-output
 ```
 
-> **Note:** this script adds a `$lib` property of `posthog-python`, overriding any `$lib` property already set.
+Notes:
+- this script adds a `$lib` property of `posthog-python`, overriding any `$lib` property already set.
+- if the script fails for some reason just run it again with the latest cursor, there are some transient issues that can happen. 
+
+## Switching tracking in you product
 
-## Option 2 for events migration: the replicator
+To make sure we don't miss any events we'll first want to do these two steps, so that we can switch tracking in our product without losing any events:
 
-To migrate your events, we can use the [PostHog Replicator app](/docs/apps/replicator), which enables us to stream both incoming and historical events to PostHog Cloud
+1. enable [PostHog Replicator app](/docs/apps/replicator) (with geoip disabled setting) to forward events from your self-hosted instance to your cloud instance while not all clients have updated their tracking code
```

Comment:
I'm confused, is the replicator available or not? This PR deleted some references to it, but it is still used here?

Reply:
It is only available for self-hosted?

Reply:
it's available for everyone, but it's only available for continuous export (not for historical export anymore)